### ISSUE TITLE
Exporting subscribers fails. 

### ISSUE THREAD

**Issue by electronick86** [spatie/mailcoach-support#286]
Hi team!

I just discovered that there is as problem when trying to export a subscriber list that's to important. I already have the problem with 50k subscribers.

The error is something logic : `  (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Maximum execution time of 30 seconds exceeded at /var/www/html/vendor/laravel/framework/src/Illuminate/Database/DatabaseManager.php:90)`

Should this task be converted into a job?

**Comment by freekmurze** [comment]
> Should this task be converted into a job?

Yes, I'd accept a PR that does this. Because I assume that there will be some breaking changes due to added columns to track generation, this PR should be set to the `v4` branch.

 If none such a PR comes in, I'll take care of it myself in the coming months.

**Comment by freekmurze** [comment]
@electronick86 In mailcoach 3.9.1 we've added this fix: https://github.com/spatie/laravel-mailcoach/blob/9693f13cbfa8a8f6adc4220a1731754527b42cec/src/Http/App/Controllers/EmailLists/Subscribers/SubscribersExportController.php#L19

Could you let us know if this works for you?

**Comment by electronick86** [comment]
Hello @freekmurze ; I just working on the upgrade of a mailcoach instance with 700k subscribers. I'll deploy that by the end of this week and I let you know if I'm able to export this list! Thanks!
